pkg/collect/helm.go

```
@@ -124,26 +128,31 @@ func helmReleaseHistoryCollector(releaseName string, namespace string) ([]Releas
 			ChartVersion: r.Chart.Metadata.Version,
 			AppVersion:   r.Chart.Metadata.AppVersion,
 			Namespace:    r.Namespace,
-			VersionInfo:  getVersionInfo(actionConfig, r.Name),
+			VersionInfo:  getVersionInfo(actionConfig, r.Name, collectValues),
 		})
 	}
 
 	return results, nil
 }
 
-func getVersionInfo(actionConfig *action.Configuration, releaseName string) []VersionInfo {
+func getVersionInfo(actionConfig *action.Configuration, releaseName string, collectValues bool) []VersionInfo {
 
 	versionCollect := []VersionInfo{}
 
 	history, _ := action.NewHistory(actionConfig).Run(releaseName)
```

Its not your code, but if you can return this error too, that would be great.